components/dialog/demo/dialog.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<meta http-equiv="X-UA-Compatible" content="ie=edge">
+		<meta charset="UTF-8">
+		<link rel="stylesheet" href="../../demo/styles.css" type="text/css">
+		<script src="/node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
+		<script type="module">
+			import '../../demo/demo-page.js';
+			import '../../button/button.js';
+			import '../dialog.js';
+			import '../dialog-confirm.js';
+		</script>
+		<style>
+			.d2l-typography p:first-child { margin-top: 0; }
+			.d2l-typography p:last-child { margin-bottom: 0; }
+		</style>
+	</head>
+	<body unresolved>
+
+		<d2l-demo-page title="d2l-dialog">
+
+			<h2>Dialog</h2>
+
+			<d2l-demo-snippet>
+				<d2l-button id="open">Show Dialog</d2l-button>
+				<d2l-dialog id="dialog" title="Dialog Title">
```

Comment:
Curious... since the `title` attribute is a native thing, do we end up with an unwanted tooltip? Not sure if it'd be better to be `title-text`.

Reply:
Indeed we do.  I will update.